### ISSUE TITLE
API Veolia

### ISSUE THREAD

**Issue by mqu** [sylvek/mail-veolia-wrapper#1]
bonjour,

nous disposons maintenant d'une API permettant l'accès à ces mêmes données ; WIP ...

**Comment by mqu** [comment]
avec le lien : https://developers.api.eau.veolia.fr/

**Comment by sylvek** [comment]
@mqu j'ai tenté de me créer un compte API, mais impossible. Aussi le formulaire pour poser des questions ne semble pas ouvert.

**Comment by mqu** [comment]
@sylvec : oui, c'est la misère et ca ne date pas d'hier ; et j'ai le sentiment d'être dans une impasse !

**Comment by sylvek** [comment]
ma technique fonctionne tjrs quant à elle.. mais bizarrement depuis deux jours, je reçois les mails sur mon login et non pas le mail d'alerte 🤔

**Comment by mqu** [comment]
il y a plusieurs repository sur le thème de VEOLIA : https://github.com/topics/veolia ; je vais regarder. Merci.

**Comment by sylvek** [comment]
> ma technique fonctionne tjrs quant à elle.. mais bizarrement depuis deux jours, je reçois les mails sur mon login et non pas le mail d'alerte 🤔

remettre son mail d'alerte + attendre 2j et c'est 🆗 - les alertes passées arrivent bien à posteriori.
